src/puppetlabs/ezbake/core.clj

```diff
@@ -442,6 +442,25 @@ Additional uberjar dependencies:
             (get-timestamp-string))
     lein-version))
 
+(defn generate-package-version-from-version
+  [lein-version]
+  {:pre [(string? lein-version)]
+   :post [(string? %)]}
+  (if (.endsWith lein-version "-SNAPSHOT")
+    (format "%s"
+            (str/replace lein-version #"-SNAPSHOT|-" { "-SNAPSHOT" "" "-" "."}))
+    lein-version))
+
+(defn generate-package-release-from-version
+  [lein-version]
+  {:pre [(string? lein-version)]
+   :post [(string? %)]}
+  (if (.endsWith lein-version "-SNAPSHOT")
+    (format "%s.%s"
+            (str/replace lein-version #".*-" "0.1")
```

Comment:
EZBake doesn't support different release numbers (you don't end up with like puppetserver 5.0.0-1 and puppetserver 5.0.0-2), and packaging prepends SNAPSHOT with 0.<release>SNAPSHOT to help with correct package versioning. https://github.com/puppetlabs/packaging/pull/393/files